### ISSUE TITLE
Building tracker fails

### ISSUE THREAD

**Issue by JamesOlvertone** [zlatinb/muwire#125]
Hi,

git clone https://github.com/zlatinb/muwire
cd ./muwire
./gradlew clean tracker:bootRun

results to this:

* What went wrong:
Project 'tracker' not found in root project 'muwire'.

**Comment by zlatinb** [comment]
Yes, the mwtrackerd broke when I upgraded to Gradle 7.  If you take a look at `settings.gradle` there is a TODO I've left for myself to fix it, but it's low priority.

**Comment by zlatinb** [comment]
Thinking about it more, the tracker should really be a separate project like [mucats](https://gihub.com/zlatinb/mucats) is.  That way different gradle versions can be used for the MuWire core and then for the tracker.

**Comment by JamesOlvertone** [comment]
Yes I found that line right now too.
Well I only wanted to play around with this tracker, I don't need it actually.
